homeassistant/components/tuya/select.py

```diff
@@ -218,6 +218,26 @@
             device_class=DEVICE_CLASS_TUYA_FINGERBOT_MODE,
             entity_category=ENTITY_CATEGORY_CONFIG,
         ),
+    ),    # Robot Vacuum
+    # https://developer.tuya.com/en/docs/iot/fsd?id=K9gf487ck1tlo
+    "sd": (
+        SelectEntityDescription(
+            key=DPCode.SUCTION,
+            name="Suction",
+            entity_category=ENTITY_CATEGORY_CONFIG,
+            entity_registry_enabled_default=False,
+        ),
+        SelectEntityDescription(
+            key=DPCode.MODE,
+            name="Mode",
+            entity_category=ENTITY_CATEGORY_CONFIG,
+        ),
+        SelectEntityDescription(
+            key=DPCode.DIRECTION_CONTROL,
```

Comment:
I'm not sure if this should be implemented as a select entity, to be honest. It provides steering controls (left, right, and such) for the vacuum cleaner. I don't think that a `select` provides the best user experience for such controls?